launch/go2_mocap.launch.py

```diff
@@ -13,6 +14,12 @@ def generate_launch_description():
                                     'go2_state_publisher.launch.py'
                                   ])
     
+    mocap_use_arg = DeclareLaunchArgument(
+                    'mocap_ground_truth',
```

Comment:
I feel like this variable should be renamed. I don't find it very explicit from an external user point of view.
Maybe something like `mimic_go2_odometry` or `replace_odometry`, `publish_odometry`, ... ? (with a default value set to true)